src/resources/views/manual/tutorials/qgis-getting-started.blade.php

```diff
@@ -0,0 +1,69 @@
+    </p>
+    <p>This opens the "Data Source Manager" Dialog Box.</p>
+    <p class="text-center">
+        <a href="{{asset('vendor/geo/images/manual/QGIS_3.jpeg')}}"><img src="{{asset('vendor/geo/images/manual/QGIS_3.jpeg')}}" width="100%" style="border: 1px solid #111;"></a>
+    </p>
+    <p>In the Dialog box,
+      <ul>
+        <li>select “Protocol, HTTP(s), cloud, etc” for Source Type.</li>
+        <li> Leave the Encoding to “Automatic”</li>
+        <li>In Protocol Type select “GeoJSON”</li>
+        <li> Added Volume Images API URI "http://biigle.de/api/v1/geojson/volumes/1/images</li>
+      </ul>
+    </p>
+    <p class="text-center">
+        <a href="{{asset('vendor/geo/images/manual/QGIS_4.jpeg')}}"><img src="{{asset('vendor/geo/images/manual/QGIS_4.jpeg')}}" width="100%" style="border: 1px solid #111;"></a>
+    </p>
+    <p>In the Authentication Section, Click on “+” button. This Opens Authentication Dialog Box.
+      <ul>
+        <li>The “Name” field is optional.</li>
+        <li>In the Username field type Biigle account email address. Note: replace “@” in the email with “%40”.</li>
+        <li>Then, Generate a new Biigle API token from Settings -> <a href="http://biigle.de/settings/tokens">Token page</a> and add the token in the Password field.</li>
+        <li>Click Save.</li>
+      </ul>
+    </p>
+    <p class="text-center">
+        <a href="{{asset('vendor/geo/images/manual/QGIS_5.jpeg')}}"><img src="{{asset('vendor/geo/images/manual/QGIS_5.jpeg')}}" width="100%" style="border: 1px solid #111;"></a>
+    </p>
+    <p class="text-center">
+      <a href="{{asset('vendor/geo/images/manual/QGIS_6.jpeg')}}"><img src="{{asset('vendor/geo/images/manual/QGIS_6.jpeg')}}" width="100%" style="border: 1px solid #111;"></a>
+    </p>
+    <p>Select the newly added Authentication configuration from dropdown and click Add.</p>
+    <p class="text-center">
+      <a href="{{asset('vendor/geo/images/manual/QGIS_7.jpeg')}}"><img src="{{asset('vendor/geo/images/manual/QGIS_7.jpeg')}}" width="100%" style="border: 1px solid #111;"></a>
+    </p>
+    <p>You can add an Open street Map to the layer. In the Browser Section, right click on “OpenStreetMap” under “XYZ Tiles”.</p>
+    <p class="text-center">
+      <a href="{{asset('vendor/geo/images/manual/QGIS_8.jpeg')}}"><img src="{{asset('vendor/geo/images/manual/QGIS_8.jpeg')}}" width="100%" style="border: 1px solid #111;"></a>
+    </p>
+    <p>In the Layer section, right click on “openstreetmap” and select “Move to Bottom”</p>
+    <p class="text-center">
+      <a href="{{asset('vendor/geo/images/manual/QGIS_9.jpeg')}}"><img src="{{asset('vendor/geo/images/manual/QGIS_9.jpeg')}}" width="100%" style="border: 1px solid #111;"></a>
+    </p>
+    <p class="text-center">
+      <a href="{{asset('vendor/geo/images/manual/QGIS_10.jpeg')}}"><img src="{{asset('vendor/geo/images/manual/QGIS_10.jpeg')}}" width="100%" style="border: 1px solid #111;"></a>
+    </p>
```

Comment:
I think the OpenStreeMap part doesn't have to be included here. It's nice to know but not relevant to this manual article (and it only changes the background color of the map at this resolution anyway).